src/operator/optimizer_op.cc

```diff
@@ -705,6 +707,94 @@ only the row slices whose indices appear in grad.indices are updated (for w, m a
 .add_arguments(AdamParam::__FIELDS__());
 
 
+NNVM_REGISTER_OP(nag_update)
```

Comment:
Is this operator doing the same thing as `sgd_update`?

Reply:
yes, and `mp_nag_update` is same as `mp_sgd_update` because nag without momentum and sgd without momentum is similar. you want me to reuse that? i thought it might be weird to use them here without renaming them.